src/main/webapp/jsp/instructorStudentList.jsp

```diff
@@ -1,219 +1,233 @@
-                            </div>
-                            <br>
-                            <div id="emails">
-                                
-                            </div>
-                        </div>
-                    </div>
-                </form>
-            </div>
-    
-            <jsp:include page="<%=Const.ViewURIs.STATUS_MESSAGE%>" />
-            <%
-                courseIdx = -1;
-                
-                for (CourseAttributes course : data.courses) {
-                    InstructorAttributes instructor = data.instructors.get(course.id);
-                    if(data.displayArchive || !data.isCourseArchived(course.id, instructor.googleId) ){
-                        courseIdx++;
-            %>
-
-            <div class='panel <%= data.isCourseArchived(course.id, instructor.googleId) ? "panel-default" : "panel-info" %>'>
-                <div class="panel-heading ajax_submit">
-                    <form style="display:none;" id="seeMore-<%=courseIdx%>" class="seeMoreForm-<%=courseIdx%>" action="<%=Const.ActionURIs.INSTRUCTOR_STUDENT_LIST_AJAX_PAGE%>">
-                        <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID %>" value="<%=course.id%>">
-                        <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" value="<%=data.account.googleId %>">
-                        <input type="hidden" id="numStudents-<%=courseIdx%>" value="<%=data.numStudents.get(course.id)%>">
-                    </form>
-                    <a class="btn btn-default btn-xs pull-right pull-down course-enroll-for-test"
-                                id="enroll-<%=courseIdx%>"
-                                href="<%=data.getInstructorCourseEnrollLink(course.id)%>"
-                                title="<%=Const.Tooltips.COURSE_ENROLL%>"
-                                data-toggle="tooltip" data-placement="top"
-                                <% if (!instructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT)) { %>
-                                disabled="disabled"
-                                <% } %>>
-                                    <span class="glyphicon glyphicon-list"></span> Enroll
-                    </a>
-                    <div class='display-icon pull-right'>
-                    </div>
-                    <strong>[<%=course.id%>] : </strong><%=PageData.sanitizeForHtml(course.name)%>
-                </div>
-                <div class="panel-collapse collapse">
-                <div class="panel-body padding-0">    
-            
-                </div>
-                </div>
-            </div>
-            <%
-                    out.flush();
-                    }
-                }
-            %>
-        </div>
-
-        <jsp:include page="<%=Const.ViewURIs.FOOTER%>" />
-    </body>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>

```

Comment:
It doesn't look like most of the file was changed, although git is showing that the entire file got changed. Is it possible to remove the unchanged parts of the file from the changeset?